### ISSUE TITLE
Use of testing.T not passed to the test function.

### ISSUE THREAD

**Issue by brackendawson** [Antonboom/testifylint#197]
From https://github.com/stretchr/testify/issues/1676

## ✅  Good
```go
func TestIfy(t *testing.T) {
	assert.Equal(t, 0, 0)
}
```

## ❌ Bad
```go
func TestIfy(t *testing.T) {
	u := &testing.T
	assert.Equal(u, 0, 0)
}
```

```go
func TestIfy(t *testing.T) {
	assert.Equal(nil, 0, 0)
}
```
